Brewfile.lock.json

```diff
@@ -143,11 +182,11 @@
   "system": {
     "macos": {
       "ventura": {
-        "HOMEBREW_VERSION": "4.2.2",
+        "HOMEBREW_VERSION": "4.2.5",
         "HOMEBREW_PREFIX": "/opt/homebrew",
         "Homebrew/homebrew-core": "api",
         "CLT": "15.1.0.0.1.1700200546",
-        "Xcode": "14.3.1",
+        "Xcode": "14.1",
```

Comment:
What are the reasons to downgrade Xcode version? 
Xcodegen claims to be compatible with Xcode 14.3 starting from [2.36.0](https://github.com/yonaskolb/XcodeGen/blob/master/CHANGELOG.md#2360), the project uses 2.38.0

Reply:
I've just been using Xcode 14.1 everywhere to make sure I don't miss anything on the minimum supported version